User guide: "use of undeclared crate or module `google_cloud_gax`" error when setting up dev environment

Page with issue: https://googleapis.github.io/google-cloud-rust/setting_up_your_development_environment.html

After following set up steps, `cargo build` failed with error related to unknown module:

```
error[E0433]: failed to resolve: use of undeclared crate or module `google_cloud_gax`
 --> src/main.rs:3:9
  |
3 |     use google_cloud_gax::paginator::{ItemPaginator as _, Paginator as _};
  |         ^^^^^^^^^^^^^^^^ use of undeclared crate or module `google_cloud_gax`

error[E0599]: no method named `items` found for opaque type `impl Paginator<ListSecretsResponse, ...>` in the current scope
  --> src/main.rs:12:10
   |
8  |       let mut items = client
   |  _____________________-
9  | |         .list_secrets(format!("projects/{project_id}"))
10 | |         .paginator()
11 | |         .await
12 | |         .items();
   | |         -^^^^^ method not found in `impl Paginator<ListSecretsResponse, ...>`
   | |_________|
   |
   |
  ::: .cargo/registry/src/index.crates.io-1949cf8c6b5b557f/google-cloud-gax-0.21.0/src/paginator.rs:62:8
   |
62 |       fn items(self) -> impl ItemPaginator<T, E>;
   |          ----- the method is available for `impl google_cloud_gax::paginator::Paginator<ListSecretsResponse, google_cloud_secretmanager_v1::Error>` here
   |
   = note: the full type name has been written to ''
   = note: consider using `--verbose` to print the full type name to the console
   = help: items from traits can only be used if the trait is in scope
help: trait `Paginator` which provides `items` is implemented but not in scope; perhaps you want to import it
   |
1  + use google_cloud_gax::paginator::Paginator;
   |

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `rust-user-guide-project` (bin "rust-user-guide-project") due to 2 previous errors
```

Running `cargo add google-cloud-gax` fixed the issue and the program was able to build:
```
$ cargo add google-cloud-gax
    Updating crates.io index
      Adding google-cloud-gax v0.21.0 to dependencies
             Features:
             - unstable-sdk-client
             - unstable-stream
 $ cargo build
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.24s
```

--- a/guide/src/setting_up_your_development_environment.md
+++ b/guide/src/setting_up_your_development_environment.md
@@ -76,6 +76,12 @@ Manager API, do the following:
    cargo add google-cloud-secretmanager-v1
    ```
 
+1. Add the [google-cloud-gax] crate to the new project
+
+```shell
+cargo add google-cloud-gax
+```
+
 1. Add the [tokio] crate to the new project
 
    ```shell
@@ -130,6 +136,7 @@ Note: The source of the Cloud Client Libraries for Rust is
 [authn-client-libraries]: https://cloud.google.com/docs/authentication/client-libraries
 [documentation for google cloud products]: https://cloud.google.com/products
 [google cloud cli]: https://cloud.google.com/sdk/
+[google-cloud-gax]: https://crates.io/crates/google-cloud-gax
 [rust]: https://www.rust-lang.org/
 [rust-getting-started]: https://www.rust-lang.org/learn/get-started
 [secret manager]: https://cloud.google.com/secret-manager/docs/overview

